anvil/__init__.py

```diff
@@ -38,19 +38,20 @@ class Anvil(object):
 
     """
 
-    def __init__(self, kiln_prefix):
+    def __init__(self, kiln_prefix, verifySslCert):
```

Comment:
Please change to be `verify_ssl_cert` in keeping with PEP8: https://www.python.org/dev/peps/pep-0008/.  Otherwise, things look fine.  I will admit that we no longer use kiln, so I cannot test the change myself, but I can work to publish a new version to pypi or whatever you need.